qulice-xml/src/main/java/com/qulice/xml/XmlValidator.java

```diff
@@ -53,6 +53,7 @@
  *
  * @author Yegor Bugayenko (yegor@tpc2.com)
  * @version $Id$
+ * @checkstyle IndentationCheck (5 lines)
```

Comment:
@pinaf I don't understand your comment. May be I'm wrong but I think the indentation above is correct. In contrary, I think the code in your comment doesn't comply (lcurly should be on a new line, the same for the right parenthesis). What do you think?